examples/complete/main.tf

```
@@ -1,8 +1,25 @@
 module "domain_protect" {
-  source = "../../"
+  #source = "../../"
+  source  = "domain-protect/domain-protect/aws"
+  version = "0.5.1"
```

The hope with the `examples/complete` is to run integration tests for each PR so it should continue to use the relative path than the published path.

Maybe we can do this instead?

```suggestion
  source = "../../"

  # source  = "domain-protect/domain-protect/aws"
  # version = "0.5.1"
```

---

thanks, updated